.github/workflows/labeler.yml

```
@@ -7,11 +7,12 @@ jobs:
     runs-on: ubuntu-latest
     name: Label the PR size
     steps:
-      - uses: codelytv/pr-size-labeler@v1
+      - uses: codelytv/pr-size-labeler@message
         with:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          xs_max_size: '10'
-          s_max_size: '50'
-          m_max_size: '200'
-          l_max_size: '450'
+          xs_max_size: '1'
+          s_max_size: '2'
+          m_max_size: '3'
+          l_max_size: '4'
```

Having tested out it works as expected, we should revert this configuration:

```suggestion
          xs_max_size: '10'
          s_max_size: '50'
          m_max_size: '200'
          l_max_size: '450'
```